mod_reforged/hooks/ai/tactical/behavior.nut

```diff
@@ -5,9 +5,11 @@
         if (_target.getType() == ::Const.EntityType.Mortar) return 0.01;
 
         local score = __original(_entity, _target, _skill);
-        if (_skill != null && !_skill.isRanged() && _target.getSkills().hasSkill("effects.rf_rebuke") && _target.getTile().getDistanceTo(_entity.getTile()) == 1)
+        if (_skill != null && !_skill.isRanged())
```

Comment:
I get, checking for isRanged here for performance reasons (sometimes saves us a getSkillByID search)
But it is redundant as it is also checked for in `willProc`

Reply:
Yeah I only added it for performance reason. But I agree it is redundant and I also don't like repeated code - as it makes it more prone to mistakes later when we wanna change something in one place and forget to do it in another place. I think I will remove the `isRanged()` check here and eat the performance hit. That function already has a ton of `hasSkill` checks in the vanilla function so one more check isn't a huge deal.